test/integration_test_helper.rb

```diff
@@ -47,6 +92,11 @@ class << self
   # Stop ActiveRecord from wrapping tests in transactions
   self.use_transactional_tests = false
 
+  def work_around_selenium_file_detector_bug
+    # see: https://stackoverflow.com/questions/70441796/selenium-webdriver-for-aws-device-farm-error-when-sending-period-keystroke-t
```

Comment:
If I'm really pedantic I'd suggest yardoc tags (which we don't use enough in our codebase).
```suggestion
    # @see https://stackoverflow.com/questions/70441796/selenium-webdriver-for-aws-device-farm-error-when-sending-period-keystroke-t
    def work_around_selenium_file_detector_bug
```

Reply:
Shouldn't we add a rubocop for it then?